streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/impl/KubernetesSessionClientV2.scala

```diff
@@ -133,4 +165,116 @@ object KubernetesSessionClientV2 extends KubernetesClientV2Trait with Logger {
     }
   }
 
+  private def genFlinkConfig(deployRequest: DeployRequest): Configuration = {
+    val flinkConfig =
+      extractConfiguration(deployRequest.flinkVersion.flinkHome, deployRequest.properties)
+    flinkConfig
+      .safeSet(DeploymentOptions.TARGET, KubernetesDeploymentTarget.SESSION.getName)
```

Comment:
The Flink Configuration likes `DeploymentOptions.TARGET`,`KubernetesConfigOptions.NAMESPACE`, `KubernetesConfigOptions.KUBERNETES_SERVICE_ACCOUNT`, etc  should be set into FlinkDeploymentDef object and remove them from `FlinkDeploymentDef.flinkConfiguration `

Reply:
What does it mean that these values are not needed?

Reply:
At present, the redundant parameters are taken from flinkCluster. I think this part can not be deleted and will be deleted after subsequent optimization.